conf/app.conf

```diff
@@ -20,3 +20,4 @@ staticBaseUrl = "https://cdn.casbin.org"
 isDemoMode = false
 batchSize = 100
 ldapServerPort = 389
+languages  = zh,en,fr,ru,ja,de,es,ko

```

Comment:
order should be `en,zh,...`, the same order as:

![image](https://user-images.githubusercontent.com/3787410/196762162-3acb76f7-99fd-407d-a74e-87048bdcf392.png)